### PR TITLE
Feature/secured admin area

### PR DESCRIPTION
**Pull request by jhackett1**
- secure the user manager to non-admins
- hide the nav link to the user manager for non-admins

- fix a bug with last_logged_in not displaying

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -43,4 +43,10 @@ def set_user
   def user_params
     params.require(:user).permit(:email, :admin)
   end
+
+  def require_admin
+    return if current_user.admin
```
Comment (timwis) [inline app/controllers/users_controller.rb:48]: Wouldn't this return a falsy value if the user is an admin? I would have expected `return true if current_user.admin` but maybe I'm just not used to ruby

**Comment by timwis** [comment]
LGTM apart from that one question I left

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (eddieleeper-nb) [reply app/controllers/users_controller.rb:48]: it always returns `nil` but the redirect isn't executed because the function returns

Comment (timwis) [reply app/controllers/users_controller.rb:48]: ah right, makes sense. it's "require admin" not "is admin"

**Pull request merged**